charts/platform-service/templates/_helpers.tpl

```diff
@@ -30,6 +30,16 @@ If release name contains chart name it will be used as a full name.
   {{ include "platform-service.name" . }}.{{ .Release.Namespace }}.svc.cluster.local
 {{- end -}}
 
+{{- define "platform-service.imageRepository" -}}
+{{- if .Values.image.full -}}

```

Comment:
Shouldn't it fail only if fluxAutomation.enabled? looks like now .full syntax is disabled at all times?

Reply:
possibly, though the only usage of this is when we need to be able to extract only the repository part, and that's not supported with the full syntax... I don't think this is a problem personally, but we could move this template into the place it's used instead of general if it's too specific with implicit failure modes